coala

```diff
@@ -18,10 +18,10 @@ import sys
 # Start ignoring PyImportSortBear and PycodestyleBear*, we need to assert the
 # version before importing things that could cause an ImportError
 from coalib import assert_supported_version
+from coalib.coala import main
```

Comment:
see the comment immediately above.

"we need to assert the version before importing things that could cause an ImportError"

Reply:
IMO we should keep this part as it was earlier and add `PEP8Bear` with `PyImportSortBear and PycodestyleBear*`.

Reply:
@sangamcse I tried
```
# Start ignoring PEP8Bear*, PyImportSortBear and PycodestyleBear*, we need to
# assert the version before importing things that could cause an ImportError
```
but its not working, what is the syntax expected here?